librz/core/cmd_descs/cmd_debug.yaml

```diff
@@ -195,9 +195,28 @@ commands:
             optional: true
       - name: dmw
         summary: Windows heap commands
-        cname: cmd_debug_heap_windows
-        type: RZ_CMD_DESC_TYPE_OLDINPUT
-        args: []
+        subcommands:
```

Comment:
please use `type: RZ_CMD_DESC_TYPE_ARGV_STATE` for all these subcommands... We should probably make it the default instead of `RZ_CMD_DESC_TYPE_ARGV` at some point...

Reply:
Do you plan to address this?